tests/aksperiscopeintegration_test.go

```diff
@@ -76,3 +76,14 @@ func checkifpodsrunning(t *testing.T) bool {
 
 	return false
 }
+
+func TestFilterSliceElemsWithTest(t *testing.T) {
```

Comment:
💡 please add this into a new test file `helper_test.go` since this function is test of a function in `helper` and not really an integration test.